idaes_examples/notebooks/docs/unit_models/custom_unit_models/creating_unit_model.ipynb

```diff
@@ -0,0 +1,2813 @@
+    "from idaes.core.initialization import InitializationStatus\n",
+    "from idaes.core.initialization.block_triangularization import (\n",
+    "    BlockTriangularizationInitializer,\n",
+    ")\n",
+    "from Org_property import OrgPhase   \n",
+    "from Aq_property import AqPhase\n",
+    "from liquid_extraction import LiqExtraction\n",
+    "\n",
+    "m = pyo.ConcreteModel()\n",
+    "m.fs = idaes.core.FlowsheetBlock(dynamic=False)\n",
+    "m.fs.org_properties = OrgPhase()\n",
+    "m.fs.aq_properties = AqPhase()\n",
+    "\n",
+    "m.fs.lex = LiqExtraction(\n",
+    "    dynamic=False,\n",
+    "    has_pressure_change=False,\n",
+    "    organic_property_package=m.fs.org_properties,\n",
+    "    aqueous_property_package=m.fs.aq_properties,\n",
+    ")\n",
+    "m.fs.lex.organic_inlet.flow_vol.fix(80 * pyo.units.L / pyo.units.hour)\n",
+    "m.fs.lex.organic_inlet.temperature.fix(300 * pyo.units.K)\n",
+    "m.fs.lex.organic_inlet.pressure.fix(1 * pyo.units.atm)\n",
+    "m.fs.lex.organic_inlet.conc_mass_comp[0, \"NaCl\"].fix(0 * pyo.units.g / pyo.units.L)\n",
+    "m.fs.lex.organic_inlet.conc_mass_comp[0, \"KNO3\"].fix(0 * pyo.units.g / pyo.units.L)\n",
+    "m.fs.lex.organic_inlet.conc_mass_comp[0, \"CaSO4\"].fix(0 * pyo.units.g / pyo.units.L)\n",
+    "\n",
+    "m.fs.lex.aqueous_inlet.flow_vol.fix(100 * pyo.units.L / pyo.units.hour)\n",
+    "m.fs.lex.aqueous_inlet.temperature.fix(300 * pyo.units.K)\n",
+    "m.fs.lex.aqueous_inlet.pressure.fix(1 * pyo.units.atm)\n",
+    "m.fs.lex.aqueous_inlet.conc_mass_comp[0, \"NaCl\"].fix(0.15 * pyo.units.g / pyo.units.L)\n",
+    "m.fs.lex.aqueous_inlet.conc_mass_comp[0, \"KNO3\"].fix(0.2 * pyo.units.g / pyo.units.L)\n",
+    "m.fs.lex.aqueous_inlet.conc_mass_comp[0, \"CaSO4\"].fix(0.1 * pyo.units.g / pyo.units.L)\n",
+    "\n",
+    "initializer = BlockTriangularizationInitializer()\n",
+    "initializer.initialize(m.fs.lex)\n",
+    "assert initializer.summary[m.fs.lex][\"status\"] == InitializationStatus.Ok"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7971e22d",
+   "metadata": {},
+   "source": [
+    "# 4. Model Diagnostics using DiagnosticsToolbox\n",
+    "\n",
+    "Here, during initialization, we encounter warnings indicating that variables are being set to negative values, which is not expected behavior. These warnings suggest that there may be flaws in the model that require further investigation using the DiagnosticsToolbox from IDAES.\n",
+    "\n",
+    "To proceed with investigating these issues, we need to import the DiagnosticsToolbox. We can gain a better understanding of its functionality by running the help function on it. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "27f7d6c9",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Help on class DiagnosticsToolbox in module idaes.core.util.model_diagnostics:\n",
```

Comment:
I don't think we need the entire `help` output here, just a short description of the tool and a link to the proper documentation.